runtime/common/src/lib.rs

```diff
@@ -28,13 +28,4 @@ macro_rules! prod_or_testnet_or_local_or_env {
 			$prod
 		}
 	};
-	($prod:expr, $test:expr, $local:expr, $env:expr) => {
-		if cfg!(feature = "frequency-rococo-local") {
-			core::option_env!($env).map(|s| s.parse().ok()).flatten().unwrap_or($local)
```

Comment:
This probably always fails due to wasm restrictions but we should remove this from our code and let dev make changes locally for testing